agent/token/store.go

```diff
@@ -26,6 +26,10 @@ type Store struct {
 	// access to the /v1/agent utility operations if the servers aren't
 	// available.
 	agentMasterToken string
+
+	// aclReplicationToken is a special token that's used by servers to
+	// replicate ACLs from the ACL datacenter.
+	aclReplicationToken string
```

Comment:
I'm thinking we should keep this with  "ACL" in the name in case we add other types of replication down the road. This one is pretty special since it needs to be a management token, which is super tied to ACLs.